### PR TITLE
vic-machine delete: Delete kvstores

### LINKED ISSUE #2769: vic-machine delete: kv store is left over
https://ci.vmware.run/vmware/vic/5925

### PR DESCRIPTION
**Pull request by emlin**
Fixes #2769

### CODE REVIEW

File: lib/install/management/store_files.go [hunk 1]
```diff
@@ -57,10 +58,16 @@ func (d *Dispatcher) deleteImages(conf *config.VirtualContainerHostConfigSpec) e
 			continue
 		}
 
+		// delete images subfolder
 		if _, err = d.deleteDatastoreFiles(imageDSes[0], path.Join(imageDir.Path, vsphere.StorageParentDir), true); err != nil {
 			errs = append(errs, err.Error())
 		}
 
+		// delete kvStores subfolder
+		if _, err = d.deleteDatastoreFiles(imageDSes[0], path.Join(imageDir.Path, store.KVStoreFolder), true); err != nil {
```
Comment (anchal-agrawal) [inline lib/install/management/store_files.go]: Something like
```
kvPath := path.Join(imageDir.Path, store.KVStoreFolder)
if _, err = d.deleteDatastoreFiles(imageDSes[0], kvPath, true) ...
```
would be more readable. Also for line 62.

**Comment by hmahmood** [comment]
lgtm

**Comment by gigawhitlocks** [comment]
lgtm

**Comment by gigawhitlocks** [comment]
:sheep: :it:

**Pull request merged**